### ISSUE TITLE
Text entered by the script does not trigger the following button to be enabled

### ISSUE THREAD

**Issue by fdPoornima** [appium/appium#11644]
## The problem
I have a hybrid mobile app , which I am using appium to automate.There is a button which gets enabled only if we enter some content into a text box. While the content is filled with appium, the following button doesnt get enabled. This issue is seen only in ios. In android, the same script works perfectly fine.

## Environment

* Appium version : 1.8.2-beta
* Last Appium version that did not exhibit the issue (if applicable): NA
* Desktop OS/version used to run Appium: Mac OS 10.13.6
* Node.js version (unless using Appium.app|exe): 8.11.2
* Mobile platform/version under test: ios 10.3.3
* Real device or emulator/simulator: Real device
* Appium CLI or Appium.app|exe:

## Link to Appium logs
The following button does not get enabled when appium enters the content into the text box, there s no appium error, so not enclosing logs

## Code To Reproduce Issue [ Good To Have ]
driver.findelement(By.id("input-create-group-name")).sendKeys("text");
the above action is supposed to trigger enabling of the create button in the next line, but it doesnt. If I debug the script and enter the same text manually, the button gets enabled.
 
driver.findelement(By.id("button-create-group-submit"))

**Comment by mykola-mokhnach** [comment]
The reason is the same as described in https://appiumpro.com/editions/36

the app expects some special event, which is not generated when the value is set by the automated script. Consider executing a manual script, which will emulate the expected event or changing the event detection algorithm in the app source.

Closed as not a bug.

**Comment by fdPoornima** [comment]
Adding the solution that worked here so that people who reach here with the same issue.

So, though the script induced sendkeys do not work, you can still do the sendkeys through the native ios keyboard as below.

driver.getKeyboard().sendKeys("yourtext");

However, I did also face issues to trigger the ios keyboard, driver.getkeyboard() was not bringing up the keyboard as expected. So here is the workaround that did bring up the keyboard eventually

TouchAction action = (new TouchAction(driver));
        action.press(startXCoordinates, startYCoordinates);
        action.moveTo(endXCoordinates, endYCoordinates);
        action.release();
        action.perform();
Basically I just performed a swipe on one of the text boxes so that it triggered the native ios keyboard
